server/command.go

```diff
@@ -81,6 +81,9 @@ const commonHelpText = "\n" +
 	""
 
 const sysAdminHelpText = "\n###### For System Administrators:\n" +
+	"Setup Jira plugin\n" +
```

Comment:
Just to confirm, these command are already present in the autocomplete right?

Reply:
@raghavaggarwal2308 Yes, this command is added in the autocomplete for the admin